cloud/ovirt/ovirt_host_pm.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+try:
+    import ovirtsdk4 as sdk
+    import ovirtsdk4.types as otypes
+except ImportError:
+    pass
+
+from ansible.module_utils.ovirt import *
+
+
+DOCUMENTATION = '''
+---
+module: ovirt_host_pm
+short_description: Module to manage power management of hosts in oVirt
+version_added: "2.3"
+author: "Ondra Machacek (@machacekondra)"
+description:
+    - "Module to manage power management of hosts in oVirt."
+options:
+    name:
+        description:
+            - "Name of the the host to manage."
+        required: true
+        aliases: ['host']
+    state:
+        description:
+            - "Should the host be present/absent/started/stopped/restarted"
+        choices: ['present', 'absent', 'started', 'stopped', 'restarted']
+        default: present
+    address:
+        description:
+            - "Address of the power management interface."
+    username:
+        description:
+            - "Username to be used to connect to power management interface."
+    password:
+        description:
+            - "Password of the user specified in C(username) parameter."
+    type:
+        description:
+            - "Type of the power management. oVirt predefined values are I(drac5), I(ilo), I(ipmilan), I(rsa),
+               I(bladecenter), I(alom), I(apc), I(eps), I(wti), I(rsb), but user can have defined custom type."
```

Comment:
Please add all fence agents supported in 4.0:

apc, apc_snmp, bladecenter, cisco_ucs, drac5, drac7 ,eps, hpblade, ilo, ilo2, ilo3, ilo4, ilo_ssh, ipmilan, rsa, rsb, wti